apps/condo/domains/common/components/TableFilter.tsx

```diff
@@ -79,14 +90,41 @@ export const getTextFilterDropdown = (columnName: string, setFiltersApplied: Rea
                 // @ts-ignore
                 value={selectedKeys}
                 onChange={(e) => {
-                    // @ts-ignore
-                    setSelectedKeys(e.target.value)
-                    setFiltersApplied(true)
-                    confirm({ closeDropdown: false })
+                    if (isFunction(setSelectedKeys)) setSelectedKeys(e.target.value)
```

Comment:
Is it ok to shorten this to `setSelectedKeys && setSelectedKeys(e.target.value)`?

Reply:
actually we want to check if setSelectedKeys not only exists, bat also is a function, that's why I always recommend using such checks in places, where code can throw exception.